public-api/src/main/resources/public-api.yaml

```diff
@@ -316,6 +316,12 @@ definitions:
       projectId:
         type: string
         description: The cloud project in which this app is running.
+      gaId:
```

Comment:
This PR does not appear to consume either of these on the client side. Should it? If not - are they actually needed here?

Reply:
The reason I added this here is due to how the server config object is rendered. It is swagger generated, so to get these values into that object, they have to be here.

Reply:
The only place which consumes the ServerConfig is our UI client code. Therefore there is no reason to change the server config unless the client is using or intends to use the new values.

Reply:
If we don't want to use this, then we will need a second mechanism for reading server-side configs. Will look into that.

Reply:
To read the value server-side, the only change that's needed is to `WorkbenchConfig`, which you've already made.